lib/ansible/cli/galaxy.py

```diff
@@ -1088,7 +1088,7 @@ def execute_init(self):
                 repository_url='http://example.com/repository',
                 documentation_url='http://docs.example.com',
                 homepage_url='http://example.com',
-                min_ansible_version=ansible_version[:3],  # x.y
```

Comment:
Is `ansible_version` always a string?

Reply:
I don't know since I haven't checked the code for that. But IIRC is the particular line with `[:3]` originating from the current devel state of the repository and not a change I made. But I might have misinterpreted your question.